### PR TITLE
P3 getting started

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR adds a Getting Started topic to the P3 docs.

### CODE REVIEW

File: docs/photoniq/p3/get-started-p3.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   - **Allow Clients To Set True Client IP Header** - No
+   - **Verification Settings** - Choose Your Own
+   - **Use SNI TLS Extension** - Yes
+   - **Match CN/SAN To** - `{{Origin Hostname}} {{Forward Host Header}} <GTM-Configured-Cname>`
+
+   The other settings in the Origin SSL Certificate Verification depend on your property certificate settings.
+
+   - **Port**
+     - **HTTP Port** - 80
+     - **HTTPS Port** - 9443
```
Comment (dwivedianurag) [inline docs/photoniq/p3/get-started-p3.md:101]: This should be 443

File: docs/photoniq/p3/p3-metrics.md [hunk 2]
```diff
@@ -30,14 +30,8 @@ In this case, the metrics would show three (3) pages and 30 requests.
...
+- **Optimized Pages**: Total pages that have been successfully optimized, indicating the scope and impact of P3's optimization efforts.
 - **Pages Failed to Optimize**: The number of pages that could not be optimized, highlighting challenges or limitations faced during the optimization process.
+- **Average Analysis Time (seconds)**: The average time taken by the Offline Analyzer to assess and formulate optimization strategies for the incoming URL which matches the configured URL pattern in policy
 - **Average Optimization Time (seconds)**: The average time taken to optimize a page, measured in seconds. This metric helps in evaluating the efficiency of the optimization process.
 - **Average Validation Time (seconds)**: The average time spent validating the optimizations applied to ensure the page doesn't visually break and to ensure the optimizations doesn't break the functionality of the page.
-- **Average Analyzer Time (seconds)**: The average time taken by the Offline Analyzer to assess and formulate optimization strategies for the incoming URL which matches the configured URL pattern in policy
```
Comment (dwivedianurag) [inline docs/photoniq/p3/p3-metrics.md:37]: All these and the below ones should be added the definitions are good

**Review by dwivedianurag — APPROVED** [review]
Ok

File: docs/photoniq/p3/p3-metrics.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/photoniq/p3/p3-metrics.md:37]: I removed them because they are not visible on the page.

**Pull request merged**